torch/nn/functional.py

```diff
@@ -4910,6 +4910,41 @@ def _mha_shape_check(query: Tensor, key: Tensor, value: Tensor,
 
     return is_batched
 
+def canonical_mask(
```

Comment:
Can we add an underscore since these are not public functions?

That is `_canonical_mask` instead of `canonical_mask`

Reply:
Done